storage/mongodb/mongo_test.go

```diff
@@ -0,0 +1,100 @@
+package mongodb_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/Lubwama-Emmanuel/Interfaces/models"
+	"github.com/Lubwama-Emmanuel/Interfaces/storage/mongodb"
+)
+
+type args struct {
+	data models.DataObject
+}
+
+func TestMongo(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		testName string
+		dbURL    string
+		args     args
+		wantErr  assert.ErrorAssertionFunc
+	}{
+		{
+			testName: "success",
+			dbURL:    "mongodb+srv://lubwamaemmanuel1:tfOkFBHXNTZHtJPq@cluster0.qubflio.mongodb.net/?retryWrites=true&w=majority", //nolint:lll
```

Comment:
After integrating viper for config management, you should look into GitHub actions secret management to protect these variables. Also, consider these breached so you should change this password